src/daemon/demo.sh

```diff
@@ -58,6 +58,12 @@ if [[ "$RGW_FRONTEND_TYPE" == "beast" ]]; then
 fi
 
 
+function fetch_initial_keys {
```

Comment:
for keyring in /var/lib/ceph/mon/*/keyring; do 
for key in $(ceph -n mon. -k $keyring  auth list |& grep ^client); do
 ceph -n mon. -k $keyring auth get "$key" -o "$key"